### PR TITLE
Refactor commands

### PR DESCRIPTION
**Pull request by mohamedhayibor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,43 @@
...
+## Usage
 
-bikes seaport
+On your terminal:
+```
+Usage: bikes                          # Display available bikes/slots for your default station
+   or: bikes <station-id>             # Display available bikes/slots for station by id
+   or: bikes <station-alias>          # Display available bikes/slots for station by custom name
+   or: bikes set-city <service-id>    # Set your default city
+   or: bikes set <param> <station-id>
```
Comment (courajs) [inline README.md]: I would change `<param>` to `<alias>` to match `<station-alias>` above

File: cli.js [hunk 2]
```diff
@@ -1,52 +1,30 @@
...
-    Usage: bikes                          # Display available bikes/slots for your default station
-       or: bikes <station-id>             # Display available bikes/slots for station by id
-       or: bikes <station-alias>          # Display available bikes/slots for station by custom name
-       or: bikes set-city <service-id>    # Set your default city
-`);
+Usage: bikes                          # Display available bikes/slots for your default station
+   or: bikes <station-id>             # Display available bikes/slots for station by id
+   or: bikes <station-alias>          # Display available bikes/slots for station by custom name
+   or: bikes set-city <service-id>    # Set your default city
+   or: bikes set <param> <station-id>
```
Comment (courajs) [inline cli.js]: If you change `<param>` above, make sure it matches here.

File: cli.js [hunk 3]
```diff
@@ -1,52 +1,30 @@
 #!/usr/bin/env node
 'use strict';
 
-const Configstore = require('configstore');
 const meow = require('meow');
-const pkg = require('./package.json');
-const cityIsValid = require('./lib/city-is-valid');
-
-const conf = new Configstore(pkg.name);
+const getStationData = require('./lib/get-station-data');
```
Comment (courajs) [inline cli.js]: Looks like this isn't used, so it can be removed

**Comment by courajs** [comment]
This mentions default station behavior but hasn't added it. I think that's probably fine for now, we can address it with a separate PR

File: README.md (hunk 1 above)
Comment (mohamedhayibor) [inline README.md]: nice catch 👍

**Comment by mohamedhayibor** [comment]
Nice catch on the station-alias naming issue and the unused global variable. Take a look and let me if there is anything else you'd like me to change ☺️

**Comment by courajs** [comment]
Looks good for now!

**Pull request merged**